elliott/elliottlib/bzutil.py

```diff
@@ -97,6 +97,8 @@ def product(self):
 
     @property
     def cve_id(self):
+        if self.bug.fields.issuetype.name == "Vulnerability":
```

Comment:
Would like to have a method for this check, something like self.is_type_vulnerability()

Reply:
we don't need a special function as this is already straightforward

Reply:
maybe I misunderstand, if you are saying self.is_type_vulnerability() is to get true/false of self.bug.fields.issuetype.name == "Vulnerability" then it possible, but this place is already in the original Bug class, the function has to be in jirabug class